src/components/table-header.tsx

```diff
@@ -105,26 +186,57 @@ const TableHeader = ({ config }: TableHeaderProps): JSX.Element => {
         </span>
       </div>
       <div className='col-xs-12 col-sm-6'>
-        <div className='pull-right'>
-          {/* <button ng-if='showAddRecord()'
-                className='chaise-table-header-create-link chaise-btn'
-                ng-className='vm.config.displayMode === recordsetDisplayModes.fullscreen ? 'chaise-btn-primary': 'chaise-btn-secondary''
-                ng-click='addRecord()' tooltip-placement='bottom-right' uib-tooltip='Create new {{vm.displayname ? vm.displayname.value : vm.reference.displayname.value}}'>
-                <span className='chaise-btn-icon glyphicon glyphicon-plus'></span>
-                <span>{{config.displayMode === recordsetDisplayModes.fullscreen ? 'Create' : 'Create new'}}</span>
-            </button>
-            <button ng-if='vm.config.displayMode === recordsetDisplayModes.fullscreen && canUpdate()'
-                className='chaise-table-header-edit-link chaise-btn chaise-btn-primary'
-                ng-click='editRecord()' ng-disabled='{{vm.pageLimit > vm.RECORDEDIT_MAX_ROWS}}' tooltip-placement='bottom-right'
-                uib-tooltip='{{ ((vm.pageLimit > vm.RECORDEDIT_MAX_ROWS) ? 'Editing disabled when items per page > ' + vm.RECORDEDIT_MAX_ROWS : 'Edit this page of records.' ) }}'>
-                <span className='chaise-btn-icon glyphicon glyphicon-pencil'></span>
-                <span>Bulk Edit</span>
-            </button> */}
+        <div
+          className='chaise-table-header-buttons'
+        >
+          {shouldShowCreateButton() && (
+            <OverlayTrigger
+              placement='bottom-end'
+              overlay={
+                <Tooltip>
+                  Create new{' '}
+                  <DisplayValue value={reference.displayname} />
+                </Tooltip>
+              }
+            >
+              <span className='chaise-table-header-buttons-span'>
+                <Button
+                  className={`chaise-btn  ${config.displayMode === RecordsetDisplayMode.FULLSCREEN ? 'chaise-btn-primary' : 'chaise-btn-secondary'}`}
+                  onClick={addRecord}
+                >
+                  <span className='chaise-btn-icon fa-solid fa-plus' />
+                  <span>{config.displayMode === RecordsetDisplayMode.FULLSCREEN ? 'Create' : 'Create new'}</span>
+                </Button>
+              </span>
+            </OverlayTrigger>
+          )}
+
+          {/* Edit Button */}
+          {shouldShowEditButton() && (
+            <OverlayTrigger
+              placement='bottom-end'
+              overlay={
+                <Tooltip>
+                  {shouldEditButtonDisabled() ? `Editing disabled when items per page > ${RECORDEDIT_MAX_ROWS}` : 'Edit this page for records.'}
+                </Tooltip>
+              }
+            >
+              <span>
+                <Button
+                  className='chaise-btn chaise-btn-primary'
```

Comment:
On master, this button has another class called `chaise-table-header-edit-link`.